src/Spryker/Zed/Install/Business/Configuration/ConfigurationInterface.php

```diff
@@ -5,35 +5,35 @@
  * For full license information, please view the LICENSE file that was distributed with this source code.
  */
 
-namespace Spryker\Install\Configuration;
+namespace Spryker\Zed\Install\Business\Configuration;
 
-use Spryker\Install\Stage\Section\Command\CommandInterface;
-use Spryker\Install\Stage\StageInterface;
-use Spryker\Style\StyleInterface;
+use Spryker\Zed\Install\Business\Stage\Section\Command\CommandInterface;
+use Spryker\Zed\Install\Business\Stage\StageInterface;
+use Spryker\Zed\Install\Communication\Style\StyleInterface;
```

Comment:
Mb we can put `StyleInterface` to the Shared layer in order to not mix business and communication layers?

Reply:
Well, we kinda can. But afaik `Shared` layer is for stuff that's meant to be used across different applications (`Yves`, `Zed`, `Client` etc), not across the layers of `Zed` application.
@stereomon what do you say?

Reply:
I think it's a good idea to move that to Shared.